testing/py3-catkin-pkg-modules/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=py3-catkin-pkg-modules
+_pkgname=catkin_pkg_modules
+pkgver=0.4.12
+pkgrel=0
+pkgdesc="Standalone Python library for the catkin build system."
+url="http://wiki.ros.org/catkin_pkg"
+arch="noarch"
+license="BSD-3-Clause"
+depends="py3-docutils py3-dateutil py3-parsing"
```

Comment:
py3-setuptools should be in makedepends= as it uses setup.py

Reply:
I will add it. I wonder how it and CI was able to build without it.

Reply:
It is problably pulled by one of the 3 depends= or one of the package one of 3 depends= depend on.

Reply:
Hmm, looks like my other package py3-empy (https://github.com/alpinelinux/aports/pull/7635) uses setup.py too but doesn't have it as a makedepend. Although empy imports disttools rather than setuptools.

Reply:
So it looks like python3 brings in a version of pip by default/accident, which includes setuptools.